lib/mutant/env/bootstrap.rb

```diff
@@ -137,7 +137,7 @@ def expression(scope)
         name = scope_name(scope) or return
 
         unless name.instance_of?(String)
-          warn("#{scope.class}#name from: #{scope.inspect} returned #{name.inspect}. #{SEMANTICS_MESSAGE}")
+          warn("#{scope.class}#name from: #{scope} returned #{name}. #{SEMANTICS_MESSAGE}")
```

Comment:
Next iterations. I saw that but deliberately ignored the possible improvement just to focus on the PRs intentions. I touched this one only for the reason of mutation coverage.